api/NICE.NextWeb.API/CacheManager/OcelotBuilderExtensions.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using CacheManager.Core;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Ocelot.Cache;
+using Ocelot.Configuration;
+using Ocelot.Configuration.File;
+using Ocelot.DependencyInjection;
+
+namespace NICE.NextWeb.API.CacheManager
+{
+    public static class OcelotBuilderExtensions
+    {
+        public static IOcelotBuilder AddCacheManager(this IOcelotBuilder builder,
```

Comment:
There's a lot in here that's basically the same as the built-in `AddCacheManager` in `OcelotBuilderExtensions` (https://github.com/ThreeMammals/Ocelot/blob/develop/src/Ocelot.Cache.CacheManager/OcelotBuilderExtensions.cs). Can we not use that? And un/re-register/replace the necessary implementations with our own? Might make ours lighter and more elegant?

Reply:
its a static class - so don't think its possible. Each IOcelotCache instance is dependant on an ICacheManager.